### PR TITLE
Iss485 - backend database

### PR DESCRIPTION
**Pull request by SandeepThokala**

### CODE REVIEW

File: batch.py [hunk 1]
```diff
@@ -102,13 +103,35 @@ def parse_args():
...
 
 
-def process_feed(args, callback=None):
+def open_connection():
+    """ open connection to database, initialize tables if they don't exist
+        :out:
+            :cursor: interactive sql object containing tables
+    """
+    conn = psycopg2.connect(host="localhost", dbname="gsaid_db", user="postgres",
+                            password="12345", port="5432")
```
Comment (GopiGugan) [inline batch.py:112]: Passowrd, database name, etc. should be an environment variable. For example, https://github.com/PoonLab/covizu/blob/4ab1ba45b919a45d974c0dc58bf64f09c7dfadb1/batch.py#L20-L23

**Comment by GopiGugan** [comment]
Lets also add `psycopg2` as a dependency in `INSTALL.md`

**Comment by GopiGugan** [comment]
Should there be an option to run the pipeline without using the database?

**Comment by ArtPoon** [comment]
> Should there be an option to run the pipeline without using the database?

Yes I prefer to keep old functionality around in case of an unanticipated use case, or we have to roll things back.  Specifying some `--option` with `action="store_true"` would do the job.

**Pull request merged**